corehq/apps/es/aggregations.py

```diff
@@ -193,7 +193,7 @@ def doc_count(self):
         return self.result['doc_count']
 
     def __getattr__(self, attr):
-        sub_aggregation = filter(lambda a: a.name == attr, self.aggregations)[0]
+        sub_aggregation = list(filter(lambda a: a.name == attr, self.aggregations))[0]
```

Comment:
should probably be rewritten as:

`next(agg for agg in self.aggregations if agg.name == attr)`

https://stackoverflow.com/a/2364277